tests/cache_manager_tests.py

```diff
@@ -1 +1,87 @@
-# white box tests for cache_manager
\ No newline at end of file
+# white box tests for cache_manager# -*- coding: utf-8 -*-
```

Comment:
Not sure this works to have the coding comment after a regular comment. Typically it would be like this:

```python
# -*- coding: utf-8 -*-

# whitebox tests for cache_manager
```